cmds/hwclock/hwclock_linux.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
```

Comment:
Entirely up to you, but if you put a build tag in here
// build +linux

then when I add the osx pipeline in azure I won't have to fix this :-)

your call, I am approving it.

Reply:
I renamed the file to `hwclock_linux.go` (like it is the case for linux specific commands like dmesg or insmod), so that should have the same effect as a build tag, right?